docs/logs/customizing-the-sdk/README.md

```diff
@@ -34,16 +34,26 @@ TODO
 
 ## Filtering LogLevels
 
-TODO
-
-### via appsettings.json
+[`ILogger`](https://docs.microsoft.com/dotnet/core/extensions/logging)
+implementations have a built-in mechanism to apply log filtering.
+This filtering lets you define the minimum
```

Comment:
we shouldn't be documenting this here. Simply linking to Ilogger docs is sufficient. (This is somewhat incorrect, as Ilogger filtering is not just based on LogLevel only. I'd just link to ilogger docs)